### DISCUSSION TITLE
No Gravity

### DISCUSSION THREAD

**Discussion by TerenceXue-tech** [isaac-sim/IsaacLab#1317] [Q&A] [answered]
The USD file I imported into the environment is not affected by gravity, even though it has mass and is defined to be subject to gravity. I can also see that the default configuration of the environment includes gravity. Why is this happening?
`        #Flange-link
    Flange_link=RigidObjectCfg(
        prim_path="/World/Spacecraft/Flange_link",
        spawn=sim_utils.UsdFileCfg(
            usd_path=r"/home/xtc/IsaacLab/source/extensions/omni.isaac.lab_assets/data/Props/falan_link/falan_link.usd",
            scale=(0.1,0.1,0.1),#还可以定义材料
            activate_contact_sensors=True,
            rigid_props=sim_utils.RigidBodyPropertiesCfg(
            disable_gravity=False,
            rigid_body_enabled=True,
            kinematic_enabled=True,)),
        init_state=RigidObjectCfg.InitialStateCfg(pos=(0.8,-0.2,0.3))
        )`
![image](https://github.com/user-attachments/assets/44638830-de0a-47de-b9d5-7ee7a93bbbfb)

**Comment by TerenceXue-tech** [comment]
GUI shows gravity
![image](https://github.com/user-attachments/assets/c037f899-c784-48fc-a812-755dccacfad2)

**Comment by kellyguo11** [comment, marked as answer]
Hi there, please set kinematic_enabled to False. Kinematic bodies are not simulated by physics - https://docs.omniverse.nvidia.com/kit/docs/omni_physics/latest/extensions/runtime/source/omni.physx/docs/index.html#kinematic-body

**Reply by TerenceXue-tech** [reply]
Thanks a lot! It works.
